### PR TITLE
infer 1.0.0 blog post

### PR DESCRIPTION
**Pull request by simonpcouch**
A blog post for the first major release of [tidymodels/infer](https://github.com/tidymodels/infer)! This is the first tidyverse.org post on the package, so the first half introduces the big ideas of the package and the second gives an overview of the new functionality in v1.0.0.

We plan to submit to CRAN the week of July 12th—this post is tentatively dated for the 19th.

cc @mine-cetinkaya-rundel

### CODE REVIEW

File: content/blog/infer-1-0-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+obs_mean
+```
+
+The distribution here just defines the spread for the standard error calculation.
+
+```{r}
+ci <- 
+  get_confidence_interval(
+    dist,
+    level = .95,
```
Comment (mine-cetinkaya-rundel) [inline content/blog/infer-1-0-0/index.Rmd:349]:
```suggestion
    level = 0.95,
```

**Review by mine-cetinkaya-rundel — CHANGES_REQUESTED** [review]
Looks great! Some suggestions on phrasing as well as consistency of language particularly regarding bootstrapping not coming after hypothesize for confidence intervals.

Date should also be changed to a date after CRAN release, that can happen when CRAN release goes through.

**Comment by simonpcouch** [comment]
The draft is much clearer with your edits--thank you🦋

**Review by mine-cetinkaya-rundel — APPROVED** [review]
Looks great with the changes in place!

Approving the PR now but one pending item before publishing will be to adjust the blog post date based on CRAN release date + preferences (if any) from the tidyverse team.

**Comment by simonpcouch** [comment]
Submitted to CRAN! Will update the post's release date upon acceptance.

**Comment by simonpcouch** [comment]
v1.0.0 was accepted to CRAN!🎉 Will change the date of the post to this Monday the 16th.

File: content/blog/infer-1-0-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're super excited announce the release of [infer](https://infer.tidymodels.org/) 1.0.0! infer is a package for statistical inference that implements an expressive statistical grammar that adheres to the tidyverse design framework. Rather than providing methods for specific statistical tests, this package consolidates the principles that are shared among common hypothesis tests and confidence intervals into a set of four main verbs (functions), supplemented with many utilities to visualize and extract value from their outputs.
```
Comment (hadley) [inline content/blog/infer-1-0-0/index.Rmd:33]: Is it worth a sentence somewhere that this is specifically designed for teaching?

File: content/blog/infer-1-0-0/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,429 @@
+---
+output: hugodown::hugo_document
+
+slug: infer-1-0-0
+title: infer 1.0.0
+date: 2021-08-16
```
Comment (hadley) [inline content/blog/infer-1-0-0/index.Rmd:6]: Needs an update

File: content/blog/infer-1-0-0/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+  calculate(stat = "diff in means")
+```
+
+or
+
+```{r, error = TRUE}
+gss %>%
+  specify(college ~ partyid, success = "degree") %>%
+  calculate(stat = "diff in props")
+```
```
Comment (hadley) [inline content/blog/infer-1-0-0/index.Rmd:389]: This (and below) needs to be indented to correctly line up the bullets.

**Review by hadley — APPROVED** [review]

**Pull request merged**